homeassistant/components/lovelace/__init__.py

```diff
@@ -249,3 +260,25 @@ def _register_panel(hass, url_path, mode, config, update):
         kwargs["sidebar_icon"] = config.get(CONF_ICON, DEFAULT_ICON)
 
     frontend.async_register_built_in_panel(hass, DOMAIN, **kwargs)
+
+
+async def _create_map_dashboard(hass: HomeAssistant):
+    translations = await async_get_translations(
+        hass, hass.config.language, "dashboard", {onboarding.DOMAIN}
```

Comment:
Why does this translation live in `onboarding` and not in `map`, which is doing the work ?